sklearn/metrics/pairwise.py

```diff
@@ -1187,29 +1187,32 @@ def linear_kernel(X, Y=None, dense_output=True):
 
 
 def polynomial_kernel(X, Y=None, degree=3, gamma=None, coef0=1):
-    """
-    Compute the polynomial kernel between X and Y::
-
+    """Compute the polynomial kernel between X and Y.
         K(X, Y) = (gamma <X, Y> + coef0)^degree
```

Comment:
This is treated as one continuous summary entry, hence the current failing test:
```
E            - SS03: Summary does not end with a period
E            - SS06: Summary should fit in a single line
```
We can also give it math highlighting/formatting.

```suggestion
    """
    Compute the polynomial kernel between X and Y.

    :math:`K(X, Y) = (gamma <X, Y> + coef0)^degree`
```